qcodes/instrument_drivers/yokogawa/GS200.py

```diff
@@ -1,5 +1,155 @@
-from qcodes.instrument.visa import VisaInstrument
-from qcodes.utils.validators import Numbers
+from functools import partial
+
+from qcodes import VisaInstrument, InstrumentChannel
+from qcodes.utils.validators import Numbers, Bool, Enum, Nothing, Ints, Validator
+
+
+def float_round(val):
+    """
+    Rounds a floating number represented as a string
+
+    Parameters
```

Comment:
@sohailc  Qcodes uses [google](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) python doc style and not [numpy doc](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html#example-numpy).

Reply:
Oh begging your pardon. I was under the impression that I was using the google python doc style. I will review the style an correct accordingly

Reply:
On that note, I think that all core devs are supposed to use type hints.

Reply:
I have never used type hints so I still need to learn to use them. I am sure its not rocket science though :-) I will use type hints from now on